tests/integration/test_dictionaries_with_bad_structure/configs/dictionaries/bad_dict.xml

```diff
@@ -0,0 +1,37 @@
+    <clickhouse>
```

Comment:
It make sense to rename this file into `invalid_dict`.

Reply:
Done